src/main/java/frc/robot/RobotContainer.java

```diff
@@ -6,6 +6,7 @@
 
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
```

Comment:
Didn't you say we are using version 2 and not j?

Reply:
Notice on line 11 it says wpilibj2 for "command"? Only the "command" library has a V2